app/_src/gateway-operator/customization/pod-disruption-budget.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Defining PodDisruptionBudget for DataPlane
+---
+
+`DataPlane` resources can be configured with a `PodDisruptionBudget` to control:
+- The number or percentage of pods that can be unavailable during maintenance (e.g. rollout).
+- The number or percentage of pods that has to be available during maintenance (e.g. rollout).
+
+This is useful when you want to ensure that a certain number or percentage of pods are always available.
+
+See the [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) guide
+for more details on `PodDisruptionBudget` API itself.
+
+## Creating DataPlane with PodDisruptionBudget
```

Comment:
Perhaps it would be worth adding an example for `Gateway` as well? Either as another subsection or changing this to have 2 tabs with snippet: 1 for `DataPlane`, 1 for `Gateway` like in https://github.com/Kong/docs.konghq.com/blob/67445da60c9aae1327a8a7c6074354ecd3c5e79b/app/_hub/kong-inc/acl/how-to/_index.md#L22-L58 using `navtabs`. WDYT?

Reply:
Currently, the implementation doesn't allow defining PDB from the `GatewayConfigurationSpec` level. Do you think we should cover this as part of https://github.com/Kong/gateway-operator/issues/142?

Reply:
Oh I missed that. We should add a separate task. 142 is scoped only for `DataPlane`s and that was our priority for Cloud Gateways. We can implement it for `Gateway` when there's demand.

WDYT?

Reply:
I've created an issue to track this: https://github.com/Kong/gateway-operator/issues/469